src/rules/converters/member-access.ts

```diff
@@ -1,15 +1,67 @@
 import { RuleConverter } from "../converter";
 
-export const convertMemberAccess: RuleConverter = () => {
+export enum AccessibilityLevel {
+    Explicit = "explicit", // require an accessor (including public)
+    NoPublic = "no-public", // don't require public
+    Off = "off", // don't check
+}
+
+export enum MemberAccessArguments {
+    NoPublic = "no-public",
+    Accessor = "check-accessor",
+    Constructor = "check-constructor",
+    ParameterProp = "check-parameter-property",
+}
+
+export const convertMemberAccess: RuleConverter = tslintRule => {
+    const schema: any = {
+        accessibility: AccessibilityLevel.Explicit,
+    };
+
+    if (
+        !(
+            tslintRule.ruleArguments.length === 0 ||
+            tslintRule.ruleArguments[0] === false ||
+            tslintRule.ruleArguments.length < 2
+        )
```

Comment:
The first check is a subset of the third, so you could simplify this to:

```ts
!(
    tslintRuleArguments.length < 2 ||
    tslintRuleArguments[0] === false
)
```

...which could be made a teeny bit simpler IMO by removing the `!`:

```ts
tslintRuleArguments >=2 && tslintRuleArguments[0] !== false
```